tn_api_hotel.yml

```diff
@@ -806,7 +806,22 @@ components:
     HotelMediaRequest:
         required:
           - hotels
+          - credential_info
         properties:
+            credential_info:
```

Comment:
Can we pass datasource as well - it's not 100% necessary but then it matches the CredentialInfo object that we have everywhere else.

Reply:
added as required